app-utils/activate-linux/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=activate-linux
+PKGSEC=utils
+PKGDEP="cairo x11-lib wayland"
+BUILDDEP="wayland-protocols x11-proto"
+PKGDES="\"Activate Windows\" watermark"
```

Comment:
Meme program that displays an \"Activate Windows\"-like watermark on the desktop